--- poseidonCE-2.3.1/bin/poseidon.sh	2004-04-30 19:04:26.031878472 +0000
+++ poseidonCE-2.3.1/bin/poseidon.sh.orig	2004-04-30 19:05:54.633408992 +0000
@@ -27,13 +27,18 @@
 
 cmd=`RealPath`
 echo "Cmd: $cmd"
-instdir=`dirname "$cmd"`
+#instdir=`dirname "$cmd"`
 project=$1
 loglevel=$2
 
-cd "$instdir"
+#cd "$instdir"
+
+POSEIDONCE_HOME=$HOME/.poseidon2/CE
+POSEIDONSE_HOME=$HOME/.poseidon2/SE
+  
+#CLASSPATH=../lib/floating-server.jar:../lib/poseidon.jar:../lib/umlplugin.jar
+CLASSPATH=DATADIR/poseidonCE/lib/floating-server.jar:DATADIR/poseidonCE/lib/poseidon.jar:DATADIR/poseidonCE/lib/umlplugin.jar
 
-CLASSPATH=../lib/floating-server.jar:../lib/poseidon.jar:../lib/umlplugin.jar
 CLASSPATH=$CLASSPATH:$JAVA_HOME/jre/lib/rt.jar      # for compilation
 
 export CLASSPATH
